Plugins/NativeCamera/Source/NativeCamera.cpp

```diff
@@ -66,18 +66,7 @@ namespace Babylon
 
                 // append media devices to navigator
                 Napi::Object mediaDevices = Napi::Object::New(env);
-                mediaDevices.Set("getUserMedia", Napi::Function::New(env, [](const Napi::CallbackInfo& info) {
-                    auto env = info.Env();
-                    auto deferred{ Napi::Promise::Deferred::New(env) };
-                    auto promise{ deferred.Promise() };
-
-                    auto& jsRuntime{ JsRuntime::GetFromJavaScript(env) };
-                    jsRuntime.Dispatch([deferred{ std::move(deferred) }](Napi::Env env) {
-                        deferred.Resolve(env.Null());
-                    });
-
-                    return promise;
-                }));
+                mediaDevices.Set("getUserMedia", Napi::Function::New(env, &NativeCamera::GetUserMedia, "getUserMedia"));
```

Comment:
This seems like an unrelate change?

Reply:
I was trying to satisfy the following compilation errors:
```
error C2440: 'return': cannot convert from 'initializer list' to 'facebook::jsi::Value'
napi-inl.h(1225,41): message : No constructor could take the source type, or constructor overload resolution was ambiguous
```
I put it in a more familiar shape and the errors went away, but it just occurred to me that I could just stick a `static_cast` on the promise, so that's what I'll do instead